storybook/storyLoader.js

```diff
@@ -1,13 +1,19 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
+// Auto-generated file created by react-native-storybook-loader
+// Do not edit.
+//
+// https://github.com/elderfo/react-native-storybook-loader.git
```

Comment:
I'm not sure why this changed, I think maybe our linter or pre-commit hook isn't working?